README.txt
You can use these example templates and icons for tt_products 2.8.0 or higher.

Get the next versions of tt_products from this online shop:
http://ttproducts.de

pob_ttp_marker_xml.htm: multiple language marker template sponsored by Rainer Lindner (pob.com)
products_css_de.html: CSS template by Robert Markula and translated by Masod Mohmand

The folder meerwijn contains the shop sources for the site http://meerwijn.nl .

Please join the team to further develop these templates at http://forge.typo3.org .
Send us your shop templates and icons to collect them here.

Regards,

Franz Holzinger


ChangeLog
(add new changes on top of this file)
2013-09-30 Franz Holzinger  <franz@ttproducts.de>
	* Add code based template files for responsive web design. Provided by Andrea Herzog-Kienast.

2012-07-18 Franz Holzinger  <franz@ttproducts.de>
	* use German button text "Zahlungspflichtig bestellen" instead of "Abschluss der Bestellung"

2012-05-19 Franz Holzinger  <franz@ttproducts.de>
	* Add example function scripts for jQuery and a TypoScript user function to check the contents of the basket

2011-11-19 Franz Holzinger  <franz@ttproducts.de>
	* Modification to HTML shop templates because the POST data did not get submitted. Use the getElementById to find the form.

2011-05-24 Franz Holzinger  <franz@ttproducts.de>
	* make use of new FORM_URL markers because they keep all parameters
	* add the DIBS payment script as an example
	* add more icons

2010-09-24 Franz Holzinger  <franz@ttproducts.de>
	* example template for the Payment Transactor API

2010-09-10 Franz Holzinger  <franz@ttproducts.de>
	* insert the ###ITEM_LIST_EMPTY### subparts
	* new ###TRACKING_TEMPLATE### for tt_products 2.9.1

2010-06-10 Franz Holzinger  <franz@ttproducts.de>
	* fix bug: The AGB marker is completely generated by tt_products. Change template files from (value="###PERSON_AGB###") to ("###PERSON_AGB###").

2010-05-26 Franz Holzinger  <franz@ttproducts.de>
	* remove en.locallang.xml, because the file marker/locallang.xml is part of tt_products
	* add markers for the creditpoint system to the template products_css_de.html
	* add the Meerwijn icons

2010-02-07 Franz Holzinger  <franz@ttproducts.de>
	* change in image marker for paymentlib

2009-08-11 Franz Holzinger  <franz@ttproducts.de>
	* fix bugs in templates
	* change paramter tx_ttproducts_pi1 to tt_products

2009-05-16 Franz Holzinger  <franz@ttproducts.de>
	* fix bug to translations
	* slight improvements of the template files

2009-03-31 Franz Holzinger  <franz@ttproducts.de>
	* Add languages for Brittish English, French and Spanish

2009-03-15 Franz Holzinger  <franz@ttproducts.de>
	* Add language for Macedonian

2009-01-09 Franz Holzinger  <franz@ttproducts.de>
	* Add languages for Russia, Hungary and Netherlands

2008-12-29 Franz Holzinger  <franz@ttproducts.de>

	* Initial release


scripts/payment_DIBS.php
<?php
/***************************************************************
*  Copyright notice
*
*  (c) 1999-2010 Kasper Skårhøj (kasperYYYY@typo3.com)
*  All rights reserved
*
*  This script is part of the TYPO3 project. The TYPO3 project is
*  free software; you can redistribute it and/or modify
*  it under the terms of the GNU General Public License as published by
*  the Free Software Foundation; either version 2 of the License or
*  (at your option) any later version.
*
*  The GNU General Public License can be found at
*  http://www.gnu.org/copyleft/gpl.html.
*  A copy is found in the textfile GPL.txt and important notices to the license
*  from the author is found in LICENSE.txt distributed with these scripts.
*
*
*  This script is distributed in the hope that it will be useful,
*  but WITHOUT ANY WARRANTY; without even the implied warranty of
*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
*  GNU General Public License for more details.
*
*  This copyright notice MUST APPEAR in all copies of the script!
***************************************************************/
/**
 * payment_DIBS.php
 *
 * This script handles payment via the danish payment gateway, DIBS.
 * Support: DIBS premium with Credit-cards  and Unibank Solo-E
 *
 * This script is used as a "handleScript" with the default lib/class.tx_ttproducts_basket_view.php of the Shop System.
 *
 * deprecated: Use the extensions transactor and transactor_dibs instead of this PHP script
 *
 * DIBS:	http://www.dibs.dk
 *
 * $Id$
 *
 * @author  Kasper Skårhøj <kasperYYYY@typo3.com>
 * @author  Franz Holzinger <franz@ttproducts.de>
 * @maintainer	Franz Holzinger <franz@ttproducts.de>
 *
 */


if (!is_object($pibase) || !is_object($this->cObj)  || !is_object($this->basket))	die('tt_products: $pibase and $pibase->cObj must be objects!');


// $lConf = $this->basketExtra["payment."]["handleScript."];
// Loads the handleScript TypoScript into $lConf.
$lConf = $confScript;


$localTemplateCode = $this->cObj->fileResource($lConf['templateFile'] ? $lConf['templateFile'] : 'EXT:tt_products/template/payment_DIBS_template.tmpl');		// Fetches the DIBS template file
if (!is_object($basketView))	{
	$error_code = '';
	$basketView = &t3lib_div::getUserObj('tx_ttproducts_basket_view',1);
	$basketView->init ($pibase, array(), FALSE, $this->templateCode, $error_code);
}
$markerObj = &t3lib_div::getUserObj('&tx_ttproducts_marker');
$localTemplateCode = $this->cObj->substituteMarkerArrayCached($localTemplateCode, $markerObj->getGlobalMarkerArray());
$calculatedArray = $this->basket->getCalculatedArray();

$tablesObj = &t3lib_div::getUserObj('&tx_ttproducts_tables');
$order = &$tablesObj->get('sys_products_orders');

$orderUid = $order->getBlankUid();	// Gets an order number, creates a new order if no order is associated with the current session

$param = '&FE_SESSION_KEY='.rawurlencode(
$GLOBALS['TSFE']->fe_user->id.'-'.
	md5(
	$GLOBALS['TSFE']->fe_user->id.'/'.
	$GLOBALS['TYPO3_CONF_VARS']['SYS']['encryptionKey']
	)
);

$products_cmd = $pibase->piVars['products_cmd'];
$products_cmd = ($products_cmd ? $products_cmd : t3lib_div::_GP('products_cmd'));
switch($products_cmd)	{
	case 'cardno':
		$tSubpart = $lConf['soloe'] ? 'DIBS_SOLOE_TEMPLATE' : 'DIBS_CARDNO_TEMPLATE';		// If solo-e is selected, use different subpart from template
		$tSubpart = $lConf['direct'] ? 'DIBS_DIRECT_TEMPLATE' : $tSubpart;		// If direct is selected, use different subpart from template
		$content=$basketView->getView(
			$localTemplateCode,
			'PAYMENT',
			$infoViewObj,
			FALSE,
			FALSE,
			$this->basket->getCalculatedArray(),
			TRUE,
			$tSubpart
		);		// This not only gets the output but also calculates the basket total, so it's NECESSARY!
		$markerArray=array();
		$markerArray['###HIDDENFIELDS###'] =
' <input type="hidden" name="merchant"
value="'.$lConf['merchant'].'">
<input type="hidden" name="amount"
value="'.round($calculatedArray['priceTax']['total']
*100).'">
<input type="hidden" name="currency"
value="'.$lConf['currency'].'">		<!--Valuta som angivet i
ISO4217, danske kroner=208-->
<input type="hidden" name="orderid"
value="'.$this->order->getNumber($orderUid).'">
<!--Butikkens ordrenummer der skal knyttes til denne transaktion-->
<input type="hidden" name="uniqueoid" value="1">
<input type="hidden" name="accepturl"
value="https://payment.architrade.com/cgi-ssl/relay.cgi/'.$lConf['relayURL'].'&products_cmd=accept&products_finalize=1'.$param.'">
<input type="hidden" name="declineurl"
value="https://payment.architrade.com/cgi-ssl/relay.cgi/'.$lConf['relayURL'].'&products_cmd=decline&products_finalize=1'.$param.'">';

		if ($lConf['soloe'] || $lConf['direct'])	{
		$markerArray['###HIDDENFIELDS###'].= '
<input type="hidden" name="cancelurl"
value="https://payment.architrade.com/cgi-ssl/relay.cgi/'.$lConf['relayURL'].'&products_cmd=cancel&products_finalize=1'.$param.'">';
		}
		if ($lConf['direct'])	{
			$markerArray['###HIDDENFIELDS###'].= '<input
type="hidden" name="opener" value="">' .
					'<input type="hidden" name="callbackurl"
value="https://payment.architrade.com/cgi-ssl/relay.cgi/'.$lConf['relayURL'].'&products_cmd=accept&products_finalize=1'.$param.'">';
			$markerArray['###WINDOW_OPENER###'] = 'onsubmit="return
doPopup(this);" target="Betaling"'; // if this is empty then no popup window will be opened
		}

		if ($lConf['test']) {
			$markerArray['###HIDDENFIELDS###'].= '
				<input type="hidden" name="test" value="foo">
			';
		}
		if ($lConf['cardType'] && !$lConf['soloe'] && !$lConf['direct'])	{
				/*
				Examples:
					DK		  Dankort
					V-DK		Visa-Dankort
					MC(DK)	  Mastercard/Eurocard udstedt i Danmark
					VISA		Visakort udstedt i udlandet
					MC		  Mastercard/Eurocard udstedt i udlandet
					DIN(DK)	 Diners Club, Danmark
					DIN		 Diners Club, international
				*/

			$markerArray['###HIDDENFIELDS###'].= '
				<input type="hidden" name="cardtype" value="'.$lConf['cardType'].'">
			';
		}
		if ($lConf['account'])  {		// DIBS account feature
			$markerArray['###HIDDENFIELDS###'].= '
				<input type="hidden" name="account" value="'.$lConf['account'].'">
			';
		}


				// Adds order info to hiddenfields.
		if ($lConf['addOrderInfo']) {
			$theFields="";
				// Delivery info
			reset($this->address->infoArray['delivery']);
			$cc=0;
			while(list($field,$value)=each($this->address->infoArray['delivery']))		{
				$value = trim($value);
				if ($value) {
					$cc++;
					$theFields.=chr(10).'<input type="hidden" name="delivery'.$cc.'.'.$field.'" value="'.htmlspecialchars($value).'">';
				}
			}

				// Order items
			reset($this->basket->itemArray);
			$theFields.='
<input type="hidden" name="ordline1-1" value="Varenummer">
<input type="hidden" name="ordline1-2" value="Beskrivelse">
<input type="hidden" name="ordline1-3" value="Antal">
<input type="hidden" name="ordline1-4" value="Pris">
';
			$cc=1;
			$priceViewObj = &t3lib_div::getUserObj('&tx_ttproducts_field_price_view');
			// loop over all items in the basket indexed by a sorting text
			foreach ($this->basket->itemArray as $sort=>$actItemArray) {
				foreach ($actItemArray as $k1=>$actItem) {
					$cc++;
					$theFields.='
	<input type="hidden" name="ordline'.$cc.'-1" value="'.htmlspecialchars($actItem['rec']['itemnumber']).'">
	<input type="hidden" name="ordline'.$cc.'-2" value="'.htmlspecialchars($actItem['rec']['title']).'">
	<input type="hidden" name="ordline'.$cc.'-3" value="'.$actItem['count'].'">
	<input type="hidden" name="ordline'.$cc.'-4" value="'.$priceViewObj->priceFormat($actItem['totalTax']).'">';
				}
			}

			$theFields.='
<input type="hidden" name="priceinfo1.Shipping"
value="'.$priceViewObj->priceFormat($calculatedArray['shipping']['priceTax']).'">';
			$theFields.='
<input type="hidden" name="priceinfo2.Payment"
value="'.$priceViewObj->priceFormat($calculatedArray['payment']['priceTax']).'">';
			$theFields.='
<input type="hidden" name="priceinfo3.Tax"
value="'.$priceViewObj->priceFormat($calculatedArray['priceTax']['total'] - $calculatedArray['priceNoTax']['total']).'">';
			$markerArray['###HIDDENFIELDS###'].=$theFields;
		}
		$content= $pibase->cObj->substituteMarkerArrayCached($content, $markerArray);
	break;
	case 'decline':
		$markerArray=array();
		$markerArray['###REASON_CODE###'] = t3lib_div::_GP('reason');
		$content =
			$basketView->getView(
				$localTemplateCode,
				'PAYMENT',
				$infoViewObj,
				FALSE,
				FALSE,
				$this->basket->getCalculatedArray(),
				TRUE,
				'DIBS_DECLINE_TEMPLATE',
				$markerArray
			);	  // This not only gets the output but also calculates the basket total, so it's NECESSARY!
	break;
	case 'cancel':
		$content =
			$basketView->getView(
				$localTemplateCode,
				'PAYMENT',
				$infoViewObj,
				FALSE,
				FALSE,
				$this->basket->getCalculatedArray(),
				TRUE,
				'DIBS_SOLOE_CANCEL_TEMPLATE',
				$markerArray
			);	 // This not only gets the output but also calculates the basket total, so it's NECESSARY!
	break;
	case 'accept':
		$content =
			$basketView->getView(
				$localTemplateCode,
				'PAYMENT',
				$infoViewObj,
				FALSE,
				FALSE,
				$this->basket->getCalculatedArray(),
				TRUE,
				'DIBS_ACCEPT_TEMPLATE'
			);
	// This is just done to calculate stuff

			// DIBS md5 keys
		$k1=$lConf['k1'];
		$k2=$lConf['k2'];

			// Checking transaction
		$amount=round($calculatedArray['priceTax']['total'] *100);
		$currency='208';
		$transact=t3lib_div::_GP('transact');
		$md5key= md5($k2.md5($k1.'transact='.$transact.'&amount='.$amount.'&currency='.$currency));
		$authkey=t3lib_div::_GP('authkey');
		if ($md5key != $authkey)	{
			$content =
				$basketView->getView(
					$localTemplateCode,
					'PAYMENT',
					$infoViewObj,
					FALSE,
					FALSE,
					$this->basket->getCalculatedArray(),
					TRUE,
					'DIBS_DECLINE_MD5_TEMPLATE'
				);		// This not only gets the output but also calculates the basket total, so it's NECESSARY!
		} elseif (t3lib_div::_GP('orderid')!=$order->getNumber($orderUid)) {
			$content =
				$basketView->getView(
					$localTemplateCode,
					'PAYMENT',
					$infoViewObj,
					FALSE,
					FALSE,
					TRUE,
					'DIBS_DECLINE_ORDERID_TEMPLATE'
				);		// This not only gets the output but also calculates the basket total, so it's NECESSARY!
		} else {
			$markerArray=array();
			$markerArray['###TRANSACT_CODE###'] = t3lib_div::_GP('transact');

			$content =
				$basketView->getView(
					$tmp='',
					'PAYMENT',
					$infoViewObj,
					FALSE,
					FALSE,
					$this->basket->getCalculatedArray(),
					TRUE,
					'BASKET_ORDERCONFIRMATION_TEMPLATE',
					$markerArray
				);
			$error=''; // TODO
			$this->order->finalize($basketView->templateCode, $basketView, $this->basket->tt_products /* TODO */,$this->basket->tt_products_cat, $this->basket->price, $orderUid,$content,$error);  // Important: $oder->finalize MUST come after the call of prodObj->getBasket, because this function, getBasket, calculates the order! And that information is used in the finalize-function
		}
	break;
	default:
		if ($lConf['relayURL']) {
			$markerArray=array();
			$markerArray['###REDIRECT_URL###'] = 'https://payment.architrade.com/cgi-ssl/relay.cgi/'.$lConf['relayURL'].'&products_cmd=cardno&products_finalize=1'.$param;
			$content =
				$basketView->getView(
					$localTemplateCode,
					'PAYMENT',
					$infoViewObj,
					FALSE,
					FALSE,
					$this->basket->getCalculatedArray(),
					TRUE,
					'DIBS_REDIRECT_TEMPLATE',
					$markerArray
				);
		} else {
			$content = 'NO .relayURL given!!';
		}
	break;
}
?>

scripts/tt_products_funcs.php
<?php
/***************************************************************
*  Copyright notice
*
*  (c) 2012 Franz Holzinger (franz@ttproducts.de)
*  All rights reserved
*
*  This script is part of the TYPO3 project. The TYPO3 project is
*  free software; you can redistribute it and/or modify
*  it under the terms of the GNU General Public License as published by
*  the Free Software Foundation; either version 2 of the License or
*  (at your option) any later version.
*
*  The GNU General Public License can be found at
*  http://www.gnu.org/copyleft/gpl.html.
*  A copy is found in the textfile GPL.txt and important notices to the license
*  from the author is found in LICENSE.txt distributed with these scripts.
*
*
*  This script is distributed in the hope that it will be useful,
*  but WITHOUT ANY WARRANTY; without even the implied warranty of
*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
*  GNU General Public License for more details.
*
*  This copyright notice MUST APPEAR in all copies of the script!
***************************************************************/


/**
 * Part of the addons_tt_products (Addons to the Shop System) extension.
 *
 * TypoScript functions for the basket
 *
 * $Id$
 *
 * @author  Franz Holzinger <franz@ttproducts.de>
 * @maintainer	Franz Holzinger <franz@ttproducts.de>
 * @package TYPO3
 * @subpackage tt_products
 *
 */

public function user_onlyBulkilyItems ($where) {
	global $TSFE, $TYPO3_DB;

	include_once (PATH_tslib.'class.tslib_content.php');

	$cObj = &t3lib_div::makeInstance('tslib_cObj');

	$basketExt = $TSFE->fe_user->getKey('ses', 'basketExt');

	if (isset($basketExt) && is_array($basketExt))	{

		$uidArr = array();

		foreach($basketExt as $uidTmp => $tmp)	{
			if ($uidTmp != 'gift' && !in_array($uidTmp, $uidArr))	{
				$uidArr[] = intval($uidTmp);
			}
		}

		if (count($uidArr) == 0) {
			return FALSE;
		}
		$where = 'uid IN (' . implode(',', $uidArr) . ')';
	}

	$where .= $cObj->enableFields('tt_products');

	$rcArray = $TYPO3_DB->exec_SELECTgetRows('*', 'tt_products', $where);

	$bAllBukily = TRUE;
	foreach ($rcArray as $uid => $row)	{
		if (!$row['special_preparation'])	{
			$bAllBukily = FALSE;
			break;
		}
	}

	return ($bAllBukily);
}


?>

scripts/tt_products_example.js
/***************************************************************
*  Copyright notice
*
*  (c) 2011 Franz Holzinger (franz@ttproducts.de)
*  All rights reserved
*
*  This script is part of the TYPO3 project. The TYPO3 project is
*  free software; you can redistribute it and/or modify
*  it under the terms of the GNU General Public License as published by
*  the Free Software Foundation; either version 2 of the License, or
*  (at your option) any later version.
*
*  The GNU General Public License can be found at
*  http://www.gnu.org/copyleft/gpl.html.
*  A copy is found in the textfile GPL.txt and important notices to the license
*  from the author is found in LICENSE.txt distributed with these scripts.
*
*
*  This script is distributed in the hope that it will be useful,
*  but WITHOUT ANY WARRANTY; without even the implied warranty of
*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
*  GNU General Public License for more details.
*
*  This copyright notice MUST APPEAR in all copies of the script!
***************************************************************/

/**
 * Part of the addons_tt_products (Addons to the Shop System) extension.
 *
 * JavaScript File
 *
 * $Id$
 *
 * @author	Franz Holzinger <franz@ttproducts.de>
 * @package TYPO3
 * @subpackage addons_tt_products
 *
 */


// use jQuery without any conficts
jQuery.noConflict();

jQuery(document).ready(function($) {
	$("a").click(function() {
		alert("Hello world!");
	});

	$( ".dateSelector" ).datepicker();
	$( ".dateSelector" ).datepicker("show");
});











ext_emconf.php
<?php

########################################################################
# Extension Manager/Repository config file for ext "addons_tt_products".
#
# Auto generated 18-07-2012 18:29
#
# Manual updates:
# Only the data in the array - everything else is removed by next
# writing. "version" and "dependencies" must not be touched!
########################################################################

$EM_CONF[$_EXTKEY] = array(
	'title' => 'Addons to tt_products',
	'description' => 'Many translations of standard texts for shops. Additional example templates and icons for the tt_products shop system version 2.9+.',
	'category' => 'example',
	'author' => 'Franz Holzinger',
	'author_email' => 'franz@ttproducts.de',
	'shy' => '',
	'dependencies' => '',
	'conflicts' => '',
	'priority' => '',
	'module' => '',
	'state' => 'stable',
	'internal' => '',
	'uploadfolder' => 0,
	'createDirs' => '',
	'modify_tables' => '',
	'clearCacheOnLoad' => 0,
	'lockType' => '',
	'author_company' => 'ttproducts.org',
	'version' => '0.3.0',
	'constraints' => array(
		'depends' => array(
			'typo3' => '4.3.0-6.1.99',
		),
		'conflicts' => array(
		),
		'suggests' => array(
		),
	),
	'_md5_values_when_last_written' => 'a:86:{s:9:"ChangeLog";s:4:"a591";s:10:"README.txt";s:4:"482a";s:16:"en.locallang.xml";s:4:"c113";s:16:"es.locallang.xml";s:4:"3578";s:12:"ext_icon.gif";s:4:"38fc";s:17:"ext_localconf.php";s:4:"fc72";s:16:"fr.locallang.xml";s:4:"0618";s:16:"hu.locallang.xml";s:4:"5d61";s:16:"mk.locallang.xml";s:4:"9c6d";s:16:"nl.locallang.xml";s:4:"067f";s:16:"ru.locallang.xml";s:4:"6165";s:14:"doc/manual.sxw";s:4:"36d2";s:19:"doc/wizard_form.dat";s:4:"2337";s:20:"doc/wizard_form.html";s:4:"796d";s:14:"meerwijn/+.gif";s:4:"36a3";s:14:"meerwijn/_.gif";s:4:"209f";s:19:"meerwijn/basket.gif";s:4:"508c";s:30:"meerwijn/detail_cadeaubon.tmpl";s:4:"c263";s:31:"meerwijn/detail_geschenken.tmpl";s:4:"b695";s:31:"meerwijn/detail_kurkenshop.tmpl";s:4:"0fad";s:29:"meerwijn/detail_shopabox.tmpl";s:4:"21a3";s:27:"meerwijn/detail_wijnen.tmpl";s:4:"63be";s:28:"meerwijn/help-icon-hover.gif";s:4:"05b8";s:22:"meerwijn/help-icon.gif";s:4:"8f01";s:28:"meerwijn/info-icon-hover.gif";s:4:"fdbf";s:22:"meerwijn/info-icon.gif";s:4:"271c";s:17:"meerwijn/kurk.gif";s:4:"c7fb";s:17:"meerwijn/lock.gif";s:4:"80ca";s:28:"meerwijn/product_detail.tmpl";s:4:"9e4a";s:36:"meerwijn/product_proefpakketten.tmpl";s:4:"9afd";s:23:"meerwijn/producten.tmpl";s:4:"103a";s:24:"meerwijn/recycle-bin.gif";s:4:"b8a7";s:24:"meerwijn/shop-a-box.tmpl";s:4:"f580";s:31:"meerwijn/totaal_geschenken.tmpl";s:4:"15ca";s:31:"meerwijn/totaal_kurkenshop.tmpl";s:4:"1306";s:29:"meerwijn/totaal_shopabox.tmpl";s:4:"f87b";s:27:"meerwijn/totaal_wijnen.tmpl";s:4:"5ee1";s:22:"meerwijn/tracking.tmpl";s:4:"aadb";s:19:"meerwijn/unlock.gif";s:4:"12c9";s:25:"meerwijn/winkelwagen.tmpl";s:4:"ff1b";s:14:"meerwijn/x.gif";s:4:"1ccf";s:26:"meerwijn/js/FormManager.js";s:4:"3ccc";s:18:"res/icons/fe/+.gif";s:4:"36a3";s:24:"res/icons/fe/AddItem.gif";s:4:"287d";s:34:"res/icons/fe/Cart-Icon-AddItem.gif";s:4:"e76c";s:40:"res/icons/fe/Cart-Icon-AddRemoveItem.gif";s:4:"9b18";s:37:"res/icons/fe/Cart-Icon-RemoveItem.gif";s:4:"b9cc";s:26:"res/icons/fe/Cart-Icon.gif";s:4:"988a";s:27:"res/icons/fe/RemoveItem.gif";s:4:"e28f";s:18:"res/icons/fe/_.gif";s:4:"209f";s:24:"res/icons/fe/addmemo.png";s:4:"c76f";s:28:"res/icons/fe/basket-grey.gif";s:4:"ca3d";s:23:"res/icons/fe/basket.gif";s:4:"508c";s:32:"res/icons/fe/help-icon-hover.gif";s:4:"05b8";s:26:"res/icons/fe/help-icon.gif";s:4:"8f01";s:32:"res/icons/fe/info-icon-hover.gif";s:4:"fdbf";s:26:"res/icons/fe/info-icon.gif";s:4:"271c";s:21:"res/icons/fe/kurk.gif";s:4:"c7fb";s:21:"res/icons/fe/lock.gif";s:4:"80ca";s:28:"res/icons/fe/recycle-bin.gif";s:4:"b8a7";s:23:"res/icons/fe/unlock.gif";s:4:"12c9";s:18:"res/icons/fe/x.gif";s:4:"1ccf";s:24:"scripts/payment_DIBS.php";s:4:"42be";s:30:"scripts/tt_products_example.js";s:4:"cb27";s:29:"scripts/tt_products_funcs.php";s:4:"e3f1";s:25:"template/TS_setup_TTP.txt";s:4:"9f6c";s:16:"template/agb.txt";s:4:"d76f";s:38:"template/example_template_bill_de.tmpl";s:4:"9800";s:38:"template/payment_DIBS_template_uk.tmpl";s:4:"3a94";s:24:"template/paymentlib.tmpl";s:4:"dde6";s:31:"template/pob_ttp_marker_xml.htm";s:4:"5946";s:29:"template/products_css_de.html";s:4:"ec39";s:29:"template/products_css_en.html";s:4:"5b04";s:25:"template/products_de.html";s:4:"d6c1";s:31:"template/products_template.tmpl";s:4:"1aa3";s:34:"template/products_template_dk.tmpl";s:4:"f9db";s:34:"template/products_template_fi.tmpl";s:4:"9850";s:34:"template/products_template_fr.tmpl";s:4:"7107";s:40:"template/products_template_htmlmail.tmpl";s:4:"aa8a";s:34:"template/products_template_it.html";s:4:"30e6";s:34:"template/products_template_se.tmpl";s:4:"477a";s:24:"template/transactor.tmpl";s:4:"c85f";s:24:"template/tt_products.css";s:4:"d0ae";s:32:"template/tt_products_example.css";s:4:"9b02";s:37:"template/tt_products_example_css.html";s:4:"9b20";s:33:"template/tt_products_specific.css";s:4:"8702";}',
	'suggests' => array(
	),
);

?>